libs/utils/test.py

```diff
@@ -134,10 +136,27 @@ def get_multi_assert(self, experiment, task_filter=""):
         experiment.
         """
         tasks = experiment.wload.tasks.keys()
-        return SchedMultiAssert(experiment.out_dir,
+        return SchedMultiAssert(self.get_trace(experiment).ftrace,
                                 self.te.topology,
                                 [t for t in tasks if task_filter in t])
 
+    def get_trace(self, experiment):
+        if not hasattr(self, "__traces"):
+            self.__traces = {}
```

Comment:
Is that "__" intentional, a private member is not enough?
Or maybe you really want to avoid subclasses to overwrite this member?